docs/DEBUGGING.md

```diff
@@ -93,6 +93,66 @@ tail -F /run/diag.out
 
 In addition this information is provided to application instances on the device using [the diag API endpoint](./ECO-METADATA.md).
 
+## Application console
+
+A running application on an EVE device has a console for input or output. You can attach to the application console from the EVE device as a control terminal if the application (VM or Container) listens to the TTY line and communicates with the virtual console /dev/hvc0 device. For example for popular linux distributions deployed as VM application this is usually the case.
+
+First list applications consoles of all running QEMU (KVM) processes:
+
+```bash
+# eve list-app-consoles
+PID     APP-UUID                                CONS-TYPE       CONS-ID
+---     --------                                ---------       ---------
+3883    e4e2f56d-b833-4562-a86f-be654d6387ba    VM              e4e2f56d-b833-4562-a86f-be654d6387ba.1.1/cons
+4072    f6d348cc-9c31-4f8b-8c4f-a4aae4590b97    CONTAINER       f6d348cc-9c31-4f8b-8c4f-a4aae4590b97.1.2/cons
+4072    f6d348cc-9c31-4f8b-8c4f-a4aae4590b97    VM              f6d348cc-9c31-4f8b-8c4f-a4aae4590b97.1.2/prime-cons
```

Comment:
Wouuld be helpful to exolain what CONS-TYPE is here. since this uuid appears to be a container and have a CONS-TYPE of CONTAINER plus one of type VM.